libs/blocks/merch-card/merch-card.js

```diff
@@ -198,11 +198,13 @@ async function extractQuantitySelect(el) {
   if (config.length !== 2) return null;
   const attributes = {};
   attributes.title = config[0].textContent.trim();
-  const quantityValues = config[1].textContent.split(',').map((value) => value.trim())
-    .filter((value) => /^\d+$/.test(value));
-  if (quantityValues.length !== 3) return null;
+  const values = config[1].textContent.split(',')
+    .map((value) => value.trim())
+    .filter((value) => /^\d*$/.test(value))
+    .map((value) => (value === '' ? undefined : Number(value)));
+  if (![3, 4, 5].includes(values.length)) return null;
   await import('../../deps/merch-quantity-select.js');
-  [attributes.min, attributes.max, attributes.step] = quantityValues.map(Number);
+  [attributes.min, attributes.max, attributes.step, attributes['default-value'], attributes['max-input']] = values;
```

Comment:
This changes doesn't appear on https://git.corp.adobe.com/wcms/tacocat.js/pull/543/files I'm curious where this change is in tacocat?